cms/blocks.py

```diff
@@ -11,3 +12,35 @@ class ResourceBlock(blocks.StructBlock):
 
     heading = blocks.CharBlock(max_length=100)
     detail = blocks.RichTextBlock()
+
+
+class FacultyBlock(blocks.StructBlock):
```

Comment:
This should be renamed to `FacultyMemberBlock`

Reply:
If it's not too much trouble, please use `Instructor` instead of `Faculty`